patches/net/minecraft/world/entity/ai/behavior/WorkAtComposter.java.patch

```diff
@@ -0,0 +1,26 @@
+--- a/net/minecraft/world/entity/ai/behavior/WorkAtComposter.java
++++ b/net/minecraft/world/entity/ai/behavior/WorkAtComposter.java
+@@ -40,18 +_,18 @@
+ 
+         int i = 20;
+         int j = 10;
+-        int[] aint = new int[COMPOSTABLE_ITEMS.size()];
++        it.unimi.dsi.fastutil.objects.Reference2IntMap<Item> amounts = new it.unimi.dsi.fastutil.objects.Reference2IntOpenHashMap<>(4);
```

Comment:
We should have some comment for why we are using a default size of `4`, especially given `COMPOSTABLE_ITEMS` has a size of two.

Reply:
I chose 4 since it's already unlikely that mods will make that value true, so I think that 4 is about the amount of villager compostables items in a pack (citation needed)